scripts/tests/chiptest/__init__.py

```diff
@@ -160,6 +160,60 @@ def _GetInDevelopmentTests() -> Set[str]:
     }
 
 
+def _GetDarwinFrameworkToolUnsupportedTests() -> Set[str]:
+    """Tests that fail in darwin-framework-tool for some reason"""
+    return {
+        "DL_LockUnlock",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_AddNewFabricFromExistingFabric",  # darwin-framework-tool does not support the GetCommissionerRootCertificate command.
+        # The name of the arguments once converted differs for chip-tool and darwin-framework-tool (attribute-ids vs attribute-id)
+        "TestAttributesById",
+        "TestBasicInformation",  # darwin-framework-tool does not support writing readonly attributes by name
+        "TestClusterComplexTypes",  # Darwin framework has no way to represent a present but null optional nullable field.
+        # When reading TestFabricScoped, the result differs because of missing fields that has been declared in the YAML step with null value to workaround some limitation of the test harness (#29110)
```

Comment:
That said, it's odd this passes in other harnesses.....

Reply:
I agree we should investigate...

Reply:
I have added some comment in https://github.com/project-chip/connectedhomeip/issues/29110#issuecomment-1928014730 to remember to investigate that too when fixing #29110